dto/src/main/java/org/jboss/pnc/dto/notification/GenericSettingNotification.java

```diff
@@ -18,28 +18,22 @@
  */
 package org.jboss.pnc.dto.notification;
 
+import org.jboss.pnc.dto.PncStatus;
 import org.jboss.pnc.enums.JobNotificationProgress;
 import org.jboss.pnc.enums.JobNotificationType;
 
 public class GenericSettingNotification extends Notification {
 
-    public static final String ANNOUNCEMENT_BANNER_CHANGED = "NEW_ANNOUNCEMENT";
-    public static final String MAINTENANCE_STATUS_CHANGED = "MAINTENANCE_STATUS_CHANGED";
+    public static final String PNC_STATUS_CHANGED = "PNC_STATUS_CHANGED";
 
-    public static GenericSettingNotification newAnnoucement(String message) {
-        return new GenericSettingNotification(ANNOUNCEMENT_BANNER_CHANGED, "{\"banner\": \"" + message + "\"}");
+    public static GenericSettingNotification pncStatusChanged(PncStatus pncStatus) {
+        return new GenericSettingNotification(pncStatus.asDto());
```

Comment:
```suggestion
        return new GenericSettingNotification(JsonUtils.toJson(pncStatus));
```

Reply:
I was wondering whether this is not what I want, and if you suggest it, turns out that's exactlyy what we want to use! :heart_eyes: